docs/manual/common/guide/production/EnterpriseSuite.md

```diff
@@ -0,0 +1,42 @@
+# Using Lightbend Enterprise Suite with Lagom
+
+Lagom is part of the [Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform)—an application development solution for building Reactive systems on the JVM powered by an open source core and the commercial [Enterprise Suite](https://www.lightbend.com/products/enterprise-suite). Lagom services can be developed, deployed and run in production using only open source components, however we recommend considering the advanced features provided by Enterprise Suite.
```

Comment:
we are missing space around `—`, just after the link [Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform)

Reply:
That wasn't an accident, I usually don't put spaces around dashes, although I acknowledge that this is a subject of some debate.

http://www.thepunctuationguide.com/em-dash.html

> The em dash is typically used without spaces on either side, and that is the style used in this guide. Most newspapers, however, set the em dash off with a single space on each side.
>
> > Most newspapers — and all that follow AP style — insert a space before and after the em dash.

https://www.quora.com/Should-you-have-a-space-around-an-em-dash

> The reason you've seen both of these used in legitimate places is that there's no definitive, "standard" answer to this question; it's a matter of style. As such, the obvious answer here is consult your house style manual. If you don't have a house style manual, consult your preferred style manual (e.g. AP, Chicago, MLA, etc.; I like Garner's Modern American Usage, which at least tries to justify its suggestions with reasons it prefers X over Y.)

http://www.sussex.ac.uk/informatics/punctuation/hyphenanddash/dash

> There are two slightly different conventions for using a dash. The more modern one is to put white spaces at both ends of a dash, while the older style uses no white spaces at all, but writes the dash solid next to whatever precedes and follows it. Both conventions are in use, and hence you may see either of the following:
>
> > The Serbs want peace — or so they say.
> > The Serbs want peace—or so they say.

https://en.wikipedia.org/wiki/Dash#Spacing_and_substitution

> According to most American sources (such as The Chicago Manual of Style) and some British sources (such as The Oxford Guide to Style), an em dash should always be set closed, meaning it should not be surrounded by spaces. But the practice in some parts of the English-speaking world, including the style recommended by The New York Times Manual of Style and Usage for printed newspapers and the AP Stylebook, sets it open, separating it from its surrounding words by using spaces or hair spaces (U+200A) when it is being used parenthetically.[39][40] The AP Stylebook rejects the use of the open em dash to set off introductory items in lists. Some writers, finding the em dash unappealingly long, prefer to use an open-set en dash. This "space, en dash, space" sequence is also the predominant style in German and French typography. (See En dash versus em dash below.)

I'm enough of a typography nerd to have an opinion on this, but I don't feel strongly enough to argue my preference too much. As far as I know, Lightbend has no official style guide to settle this so maybe @rstento can offer a suggestion. Maybe I should just change it to a colon! 😁

Reply:
I didn't know that this was a matter of style in the English language. I'm sorry, I didn't want to open that can. 

Maybe, for non-native speakers (readers), this reads quite strange because of our biases from our own languages. To be honest, I can't recall having ever seen this in any other language.  

Anyway, if that's not a syntax rule, but a style concern, we should not care much unless we have a style guide defined.

Reply:
OK, well if you think this is less clear for non-native English speakers, then it's worth changing. We certainly have many non-native readers in our audience, and our documentation should be as clear as possible. This is something I'm a bit oblivious about so I appreciate the feedback.

Reply:
I always favored the newspaper style with spaces because I thought it was easier to read. In former companies I was forced to use no spaces because of a style guide, but I agree that we should use what does not confuse non-native English speakers (readers).

Reply:
Which happens to be my preference anyway!

Reply:
OK thanks. Done!